mlrun/template_config.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import mlrun.common.schemas
+
+# These are alert templates that come built-in with the system and pre-populated on system start
+# If we define new system templates, those should be added here
+
+pre_defined_templates = [
```

Comment:
When I suggested adding them to the config, I did mean the actual mlconf. As the templates can be represented by a simple json, we can allow the API to be configured with templates before deployment and not necessarily hard-coded.

Even if that isn't what we want, I'd prefer this be part of the Config class and not in a global `templates_config.py` file only for alert templates.